client/utils/account-fees.tsx

```diff
@@ -74,28 +82,33 @@ const getFeeDescriptionString = ( fee ) => {
 	return '';
 };
 
-export const getCurrentFee = ( accountFees ) => {
+export const getCurrentFee = (
+	accountFees: FeeStructure
+): BaseFee | DiscountFee => {
```

Comment:
I believe it is not possible to specify return type only as `DiscountFee`. Since this function can return `BaseFee` or `DiscountFee`, and `BaseFee` does not have all the properties that `DiscountFee` has. Please let me know if maybe I am missing something 😅

Reply:
Corect!